OSMGeoweek-Nov-2017/README.md

```diff
@@ -0,0 +1,54 @@
+# OSMGeoweek Mapping party 
+*Mapbox, Bengaluru, 17 Nov 2017*
+
+#### Presented by:
+
+**Oormila Vinod** - oormila@mapbox.com
+
+**Chethan Gowda** - chethan@mapbox.com
+
+| | | 
+|---|---|
+|![gxb-500px-openstreetmap_logo](https://user-images.githubusercontent.com/6770741/28453477-34d856f8-6e15-11e7-95d1-022ab54b0355.png) | ![mapillary_logo svg](https://user-images.githubusercontent.com/6770741/32936973-6cc75bf6-cb9c-11e7-8701-83b2d155939c.png)|
+
+---
+
+- [OSMGeoweek.org](http://osmgeoweek.org/)
+- [Event page](https://www.mapbox.com/events/osm-geoweek-blr/)
+- [Facebook page](https://www.facebook.com/events/1239288609549971/)
+
+---
+
+## Getting started
+
+- Sign up [here](https://www.openstreetmap.org), if you are just getting started with OpenStreetMap.  
+- Navigate to [Jayanagar, Bengaluru](https://www.openstreetmap.org/#map=15/12.9390/77.5859) locality to get started with mapping using [Mapillary](https://www.mapillary.com/). 
+
+## Introduction to mapping using iD editor
+
```

Comment:
Suggestion: Will it be a good idea to show an image from Mapillary and highlight the possible features that can be mapped? It'll give folks a sense of how this can be used.

Reply:
On it. I will add it.